module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -908,16 +942,32 @@ protected function getDateTimeFromString(string $str): DateTime
     }
 
     /**
-     * Support method for getHolding(): obtaining the Due Date from OKAPI
-     * by calling /circulation/loans with the item->id, adjusting the
-     * timezone and formatting in universal time with or without due time
+     * Support method for getHolding(): obtaining the Due Date from the
+     * current loan, adjusting the timezone and formatting in universal
+     * time with or without due time
      *
-     * @param string $itemId   ID for the item to query
+     * @param string $loan     The current loan
```

Comment:
$loan is not a string here. Maybe for back-compatibility it would make sense to support `string|\stdClass` and if it's a string, look up the object.

Reply:
Agreed, I built the back compatibility.